src/stylesheets/core/_functions.scss

```diff
@@ -809,74 +815,36 @@ Gets a valid USWDS flex value
 
 /*
 ----------------------------------------
-font()
+scale()
 ----------------------------------------
-Gets
-- 'family'
-- 'size'
-- 'weight'
-based on on props passed
+Get a normalized typescale from a family
+and scale
 ----------------------------------------
 */
 
-@function font($props...){
-  $type: smart-quote(nth($props, 1));
-  @if $type == 'family' {
-    $our-family: smart-quote(nth($props, 2));
-    @if not map-has-key($project-font-stacks, $our-family){
-      @error "#{$our-family} is not a valid font family. Valid values: #{map-keys($project-font-stacks)}";
-    }
-    @elseif not map-get($project-font-stacks, $our-family) {
-      @error "Font family `#{$our-family}` is disabled in your project's theme settings. Set its value to `true` to use this family.";
-    }
-    @else {
-      @return map-get($project-font-stacks, $our-family);
-    }
+@function scale($props...){
+  $props: unpack($props);
+  @if length($props) != 2 {
+    @error "Font size requires a valid FAMILY and SIZE in the form `font('FAMILY', 'SIZE')`";
   }
-  @elseif $type == 'size' {
-    @if length($props) != 3 {
-      @error "Font size requires a valid FAMILY and SIZE in the form `font('family', 'FAMILY', 'SIZE')`";
-    }
-    $our-family: smart-quote(nth($props, 2));
-    $our-scale: smart-quote(smart-quote(nth($props, 3)));
-    @if not map-has-key($project-cap-heights, $our-family) {
-      @error "#{$our-family} is not a valid font family. Valid values: #{map-keys($project-cap-heights)}";
-    }
-    @elseif not map-get($all-type-scale, $our-scale) {
-      @error "`#{$our-scale}` is not a valid font scale. Valid values: #{map-keys($all-type-scale)}";
-    }
-    @else {
-      $this-cap: map-get($project-cap-heights, $our-family);
-      $this-scale: map-get($all-type-scale, $our-scale);
-      @if $this-scale and $this-cap {
-        @return type-scale($this-cap, $this-scale);
-      }
-      @else {
-        @error "The scale `#{$our-scale}` is disabled in your project's theme settings. Set its value to `true` to use this family.";
-      }
-    }
+  $our-family: smart-quote(nth($props, 1));
+  $our-scale: smart-quote(smart-quote(nth($props, 2)));
```

Comment:
Where is the `smart-quote` fn coming from? I cannot find it anywhere in bourbon, the sass reference, or this repo. Asking because the double-nesting seems weird to me, but I don't really have any context :D

Reply:
is it this https://github.com/uswds/uswds/blob/release-2.0/src/stylesheets/core/_functions.scss#L96 ?

Reply:
ah yep, I was searching the wrong branch. thanks!

Reply:
The double-nesting does seem redundant

Reply:
But the intention of `smart-quote()` is to format specific types in a specific way (many of these functions can accept a range of types) — and to avoid the errors that come from doing something like quoting a number or a bool.